uisdk/src/androidTest/java/com/karhoo/uisdk/address/AddressRobot.kt

```diff
@@ -24,12 +24,6 @@ class AddressRobot : BaseTestRobot() {
                 text = searchTerm)
     }
 
-    fun selectFirstAddress() {
```

Comment:
Was this meant to be removed? Is it not used anywhere?

Reply:
It wasn't used.